server/src/helpers/filestore-local.js

```diff
@@ -0,0 +1,96 @@
+'use strict';
+
+const config = require('../../config/config'),
+	 fs = require('fs'),
+	 fsutil = require('./filestore-utils');
+
+module.exports = {
+	// fileName is most likely 'maps/mapID'
+	storeMapFileLocal: async (mapFileBuffer, fileName) => {
```

Comment:
Same as comment on server/src/helpers/filestore-cloud.js : line 19

Reply:
Not sure about it for local since it is quite literally writing an actual file named that.

Reply:
Similarly, we're going to be removing local storage in the nest rewrite, this is only backwards compat for now.